README.md

```diff
@@ -74,24 +76,33 @@ If you do not wish to run from an IDE, use `gradlew.bat runClient` to run KAMI.
 
 ### Building
 #### Windows
-You can build by running `autobuild.bat` and then `autocopy.bat`, or if you prefer the manual way, do the following:
-
+You can build by running these commands (without these <>) in a terminal with the current directory being KAMI. (EG. `cd C:\Users\Username\Downloads\KAMI`)
 ```
-gradlew.bat build
-cd build/libs
+gradlew.bat <args>
 ```
-To copy the release  jar in `build/libs` you can run `autocopy.bat` or, if you prefer doing it manually, find a file `KAMI-<minecraftVersion>-<kamiVersion>-release.jar` which you can copy to the `mods` folder of a minecraft instance that has forge installed.
+Possible arguments on Windows are `build`.
+
+To copy on windows run `autocopy.bat`
```

Comment:
If there's only one argument possible, just have the instructions be:
```
gradlew.bat build
```

Also, we're under the `Windows` section: no need to mention `To copy on windows`

Reply:
Updated